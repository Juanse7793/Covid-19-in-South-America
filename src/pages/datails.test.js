import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../Redux/configureStore';
import Details from './details';

describe('Home', () => {
  it('renders the Details page and make a snapshot', () => {
    const { asFragment } = render(
      <Provider store={store}>
        <Details />
      </Provider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
