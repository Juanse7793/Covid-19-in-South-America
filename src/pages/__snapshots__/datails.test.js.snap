// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the Details page and make a snapshot 1`] = `
<DocumentFragment>
  <div
    class="Details"
  >
    <div
      class="title-container"
    >
      <h1
        class="details-title"
      >
        Covid Stats inÂ 
        <span
          class="country-name"
        />
      </h1>
    </div>
    <div
      class="images-cont"
    />
    <div
      class="details-item-cont"
    />
  </div>
</DocumentFragment>
`;
